ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerViewModel.kt

```diff
@@ -163,7 +163,7 @@ internal constructor(
     val geometry = features[0].geometry
 
     if (geometry is Point) {
-      mapController.panAndZoomCamera(geometry.coordinates)
+      panAndZoomCamera(geometry)
```

Comment:
I think `Coordinates` is the correct type to send here, no `Geometry` (i.e. `geometry.coordinates`)

Reply:
Done